R/profile_wrapper.R

```diff
@@ -54,35 +69,42 @@ profile_wrapper <- function(mydir, model_settings){
   			  to = profile_dir, overwrite = TRUE), file = "run_diag_warning.txt")
   	message(paste0( "Running profile for ", para, ".") )
 
+    # Copy the control file to run from the copy 
+    if (!file.exists(file.path(profile_dir, "control.ss_new"))) {
+      orig_dir <- getwd()
```

Comment:
I think everything in lines 74-88 could be replaced by this one line:
`run(dir = profile_dir, exe = model_settings$exe, extras = model_settings$extras)`